app/styles/_alert_boot.scss

```diff
@@ -0,0 +1,76 @@
+.md-alert-boot {
```

Comment:
I think you can achieve most of this via the Bootstrap modal: https://getbootstrap.com/docs/3.3/javascript/#modals-examples